src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+
+      <Display />
+      <ButtonPanel />
+      {/* <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header> */}
```

Comment:
- [ ] Kindly remove the boilerplate generated by React above as it will not be of any use in your application.